mail/notmuch/Portfile

```diff
@@ -5,10 +5,11 @@ PortGroup           conflicts_build 1.0
 
 name                notmuch
 version             0.28.3
+revision            1
 categories          mail
 platforms           darwin
 license             GPL-3+
-maintainers         nomaintainer
+maintainers         {seichter.de:macports @rseichter}
```

Comment:
Can `openmaintainer` be added here?

Reply:
I had thought about it. Without openmaintainer, I need to make all changes myself, which may slow down things a bit, but ensures that I fix my own mistakes. I think this will help me learn how to do it right, wouldn't you agree?

Reply:
I would say still suggest you add openmaintainer. Major changes to the port will not be made without your agreement, but it lets others make trivial fixes quickly. We really prefer ports to be openmaintainer these days, unless there is a very good reason not to, and this does not seem to me to be one of those cases.

Reply:
Fair enough. Added.

Reply:
Thanks.